### ISSUE TITLE
Examples for Pagination don't have full Accessibility support

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus-bootstrap.trimble.com#132]
### Describe the issue

The Paginations page (https://modus-bootstrap.trimble.com/components/pagination/) is missing a few accessibility guidance notes which are covered on the Bootstrap docs (https://getbootstrap.com/docs/4.6/components/pagination/).

- The Next and Previous icons should have `aria-label="Next"` and `aria-label="Previous"`
- We should add the following:

> In addition, as pages likely have more than one such navigation section, it’s advisable to provide a descriptive aria-label for the `<nav>` to reflect its purpose. For example, if the pagination component is used to navigate between a set of search results, an appropriate label could be `aria-label="Search results pages"`.

### What operating system(s) are you seeing the problem on?

Windows
